package.sh

```diff
@@ -213,9 +214,11 @@ check_tag_exists () {
 make_dir_tree() {
     work_dir=$1
     version=$2
-    mkdir -p $work_dir/$INSTALL_ROOT_DIR/versions/$version/scripts
+
+    mkdir -p $work_dir/$INSTALL_ROOT_DIR
```

Comment:
We do plenty of `mkdir` later on without checking codes. Perhaps don't bother checking any? Not sure.